### ISSUE TITLE
compatible with vue@2.2.1 ?

### ISSUE THREAD

**Issue by pakyo-pan** [wyzantinc/vue-radial-progress#4]
**vue@2.2.1**

1. import RadialProgressBar from 'vue-radial-progress';
2. webpack

When I executed the webpack command, I got the following error:
![wechatimg2](https://cloud.githubusercontent.com/assets/13342613/25304361/4a9f2a0c-2798-11e7-8300-b508fb60fde8.jpeg)
How can I solve this problem? thank you ~

**Comment by mccallofthewild** [comment]
I had the same error, but with `es2015` instead. It's because `"main"` in the package.json is pointing at `src/main.js`, which is uncompiled. The solution is to directly route to the compiled script: 
``` javascript
import RadialProgressBar from 'vue-radial-progress/dist/vue-radial-progress.min.js'
```

**Comment by jrenton** [comment]
@mccallofthewild Right, and pointing to the minified version would alternatively break for others who have the runtime only of Vue. The trick is trying to get all possible environments playing well: Vue 1.0, Vue 2.0 runtime, Vue 2.0 standalone. I'm not sure it is totally possible, but I will try, otherwise will be forced to just create separate branches for both which I wanted to avoid.

**Comment by jdinartejesus** [comment]
@jrenton any news on this issues? I'm also having the same issue.

**Comment by seanbuscay** [comment]
Hi all.  I kept getting the es2015 version of the error that @mccallofthewild mentioned during my build process.  I found these steps as a temp workaround to make it work.  I have not had the time to look at why it works.

After `npm install` or `yarn add` of the vue-radial-progress package
Go to the package directory ( `node_modules/vue-radial-progress` )
Run `npm install` to build ( or rebuild) the package and it's dependencies within the vue-radial-progress directory.

After that my auto builds work.
Hope this helps as a work around or for someone who may want to troubleshoot the issue.
